docs/detections/rules-ui-create.asciidoc

```diff
@@ -320,6 +320,10 @@ TIP: Avoid setting long time ranges with short rule intervals, or the rule previ
 
 * Click the *View details* icon (image:images/view-details-icon.png[View details icon,16,15]) in the alerts table to view the details of a particular alert.
 
+* To resize the preview, hover between the rule settings and preview areas, then click and drag the border. You can also click the border, then the collapse icon (image:images/collapse-right-icon.png[Collapse icon,16,17]) to collapse and expand the preview.
```

Comment:
```suggestion
* To resize the preview, hover between the rule settings and preview, then click and drag the border. You can also click the border, then the collapse icon (image:images/collapse-right-icon.png[Collapse icon,16,17]) to collapse and expand the preview.
```
This initially read to me as if there were multiple preview areas. Not crucial though